dotcom-rendering/src/components/Masthead.tsx

```diff
@@ -0,0 +1,106 @@
+import { palette as sourcePalette } from '@guardian/source-foundations';
+import type { EditionId } from '../lib/edition';
+import type { NavType } from '../model/extract-nav';
+import { palette as themePalette } from '../palette';
+import { Island } from './Island';
+import { Nav } from './Nav/Nav';
+import { Section } from './Section';
+import { SubNav } from './SubNav.importable';
+import { TopBar } from './TopBar.importable';
+
+type Props = {
+	nav: NavType;
+	editionId: EditionId;
+	idUrl?: string;
+	mmaUrl?: string;
+	subscribeUrl: string;
+	discussionApiUrl: string;
+	idApiUrl: string;
+	showSubNav?: boolean;
+	isImmersive?: boolean;
+	displayRoundel?: boolean;
+	hasPageSkin?: boolean;
+	hasPageSkinContentSelfConstrain?: boolean;
+};
+
+export const Masthead = ({
```

Comment:
Do we usually do this?

Reply:
- [We do for Islands](https://assets.guim.co.uk/assets/stats/islands.html)
- Documentation alongside code is usually the best way to ensure things stay in sync, and are not lost over time. I do not know what a “Masthead” is, so it could just as well be called a Mastodon or a Masterful or a Mastermind. A JSDoc may give some insight as to why this component is named thusly.
- Only a suggestion, not a blocker 🤷